apps/vr-tests-react-components/package.json

```diff
@@ -79,6 +78,7 @@
     "@fluentui/react-breadcrumb": "*",
     "@fluentui/react-rating": "*",
     "@fluentui/react-search": "*",
-    "@fluentui/react-teaching-popover": "*"
+    "@fluentui/react-teaching-popover": "*",
+    "@fluentui/react-tag-picker": ">=9.0.0-alpha"
```

Comment:
Should this be '*' 🤔

Reply:
yup

Reply:
nope, trust the automation ;) 

<img width="422" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/364affdd-2b7b-4c5a-80a7-c7711f21b8aa">

https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/prepare-initial-release/index.ts#L216-L223

Reply:
feel free to resolve @bsunderhus to unblock merging